docs/release/3.4/release-3-4-24.md

```diff
@@ -0,0 +1,132 @@
+OSG Software Release 3.4.24
+===========================
+
+**Release Date**: 2019-02-21
+
+Summary of changes
+------------------
+
+This release contains:
+
+-   BLAHP 1.18.39: Now propagates signals down to the payload job
+-   osg-pki-tools 3.0.1: Now accepts multi-word states or provinces
+-   HTCondor-CE 3.2.1: Minor adjustments to work with new HTCondor versions (8.6.x, 8.8.x, and 8.9.x)
+-   condor-cron 1.1.4: Minor adjustments to work with new HTCondor versions (8.6.x, 8.8.x, and 8.9.x)
+-   osg-release 3.4-1: New osg-rolling repository (disabled by default) allows early access to fully tested software
+-   Upcoming Repository
+    -   [Singularity 3.0.3](https://github.com/sylabs/singularity/releases/tag/v3.0.3): Bug fix release
+        -   [Change Log](https://github.com/sylabs/singularity/blob/master/CHANGELOG.md#v303---20190121)
+    -   Limited HDFS functionality for EL6 hosts
+        -   Client Only: Mount HDFS via FUSE
+
+        !!! info
+            Upgrade to EL7 to update name nodes, data nodes, XRootD, or GridFTP hosts.
+
+These [JIRA tickets](https://jira.opensciencegrid.org/issues/?jql=project%20%3D%20SOFTWARE%20AND%20fixVersion%20%3D%203.4.24%20ORDER%20BY%20priority%20DESC%2C%20key%20DESC) were addressed in this release.
+
+!!! note "Notes"
+    -   OSG 3.4 contains only 64-bit components.
+    -   StashCache is supported on EL7 only.
+    -   xrootd-lcmaps will remain at 1.2.1-2 on EL6.
```

Comment:
Not something that needs to be changed now but just a reminder that this can be removed whenever we release the new xrootd-lcmaps

Reply:
Yup. I am aware of this. Thanks for making sure